Cannot go get the project

Hi when I try to go get the project from windows7 I receive this error;

go get github.com/konimarti/kalman
# github.com/konimarti/kalman
C:\Users\Alberto\go\src\github.com\konimarti\kalman\kalman.go:72:4: K.Clone unde
fined (type mat.Dense has no field or method Clone)

--- a/go.mod
+++ b/go.mod
@@ -1,0 +1,5 @@
+module github.com/konimarti/kalman
+
+go 1.12
+
+require gonum.org/v1/gonum v0.0.0-20190628223043-536a303fd62f

--- a/kalman.go
+++ b/kalman.go
@@ -69,7 +69,7 @@ func (u *update) Update(ctx *context, z mat.Vector) error {
 		denom.Zero()
 		K.Product(ctx.P, u.H.T(), &denom)
 	} else {
-		K.Clone(kt.T())
+		K.CloneFrom(kt.T())
 	}
 
 	// update state

